packages/editor/src/components/post-actions/actions.js

```diff
@@ -982,95 +973,96 @@ const renameTemplateAction = {
 	},
 };
 
-export function usePostActions( onActionPerformed, actionIds = null ) {
+export function usePostActions( postType, onActionPerformed ) {
```

Comment:
I liked the previous flexibility of allowing the consumer to select the actions. I think we will end up needing to retrieve only some actions in some cases. But I'm fine with using postType for now until we see if that need materializes.

Reply:
I think flexibility comes with inconsistency. I'd like to avoid it until we find our selves in a situation where we want to pick what we want. Users can still pick what they want, it's just a simple filter on top of the returned value.